sotodlib/mapmaking/utilities.py

```diff
@@ -439,8 +441,48 @@ def rangemat_sum(rangemat):
         res[i] = np.sum(ra[:,1]-ra[:,0])
     return res
 
-def find_usable_detectors(obs, maxcut=0.1):
-    ncut  = rangemat_sum(obs.flags.glitch_flags)
+def flags_in_path(
+    aman: core.AxisManager, rpath: str, sep: str = "."
+) -> bool:
+    """
+    This function allows to pull data from an AxisManager based on a path.
+    Parameters:
+        - aman: An Axis Manager object
+        - path: a string with a recursive path to extract data. The path is separated via a sep.
+                For example 'flags.glitch_flags'
+        - sep: separator. Defaults to `.`
+    """
+
+    rpath = rpath.split(sep=sep)
+    flags = aman.copy()
+    while rpath and flags is not None:
+        path = rpath.pop()
+        flags = flags[path]
```

Comment:
This function does not do its job, if, e.g. "some.long.flags" is not in aman.  You need a:
```
if path not in flags._fields:
    return False
```
or similar.  Or do you want to look up the rpath, and return False if it's there but it's None?  (The docstring isn't clear about what exactly this function is supposed to do.)

Reply:
Let me think about it and fix this